docs/runtime-contract.md

```diff
@@ -126,9 +126,9 @@ the OCI specification as long as:
   contents from a particular execution. Because containers (particularly failing
   containers) can experience frequent starts, operators or platform providers
   SHOULD limit the total space consumed by these failures.
-- A container should write its own termination message to `/dev/termination-log`
+- A container SHOULD write its own termination message to `/dev/termination-log`
```

Comment:
I would make this 'ought'.

Personally I would like to see us remove this in a future change. See discussion here: https://github.com/knative/serving/pull/4035/files#r283528430

Reply:
yep - we have no say over this. Fixed.